web/styles/dark_theme.css

```diff
@@ -521,19 +521,31 @@
         background-color: hsl(0deg 0% 0% / 20%);
     }
 
-    #search_arrows .pill,
-    #compose-direct-recipient.pill-container .pill {
-        color: inherit;
-        border: 1px solid hsl(0deg 0% 0% / 50%);
-        background-color: hsl(0deg 0% 0% / 25%);
-        font-weight: 600;
-    }
+    #searchbox {
+        .navbar-search #search_query::placeholder {
+            color: hsl(0deg 0% 100% / 50%);
+        }
 
-    #search_arrows .pill:focus,
-    #compose-direct-recipient.pill-container .pill:focus {
-        color: hsl(0deg 0% 100%);
-        border: 1px solid hsl(176deg 78% 28% / 60%);
-        background-color: hsl(176deg 49% 42% / 40%);
+        /* Light theme shows hover mostly through box-shadow,
+          and dark theme shows it mostly through changing color
+          of the search button and placeholder text. */
+        .navbar-search:not(.expanded) #search_arrows:hover {
+            /* Light theme has two different (very similar) colors in the box
```

Comment:
It would be great (in another PR) to do a roundup of similar colors like this and run them by @terpimost to see where we could consolidate and make things uniform.

Reply:
[asked here](https://chat.zulip.org/#narrow/stream/101-design/topic/search.20redesign/near/1602346)